tests/test_deserialization.py

```diff
@@ -576,7 +577,7 @@ def test_timestamp_field_deserialization(self, fmt, value, expected):
     @pytest.mark.parametrize("fmt", ["timestamp", "timestamp_ms"])
     @pytest.mark.parametrize(
         "in_value",
-        ["", "!@#", 0, -1, dt.datetime(2013, 11, 10, 1, 23, 45)],
+        ["", "!@#", -1, dt.datetime(2013, 11, 10, 1, 23, 45)],
```

Comment:
Nit: might be worth testing None here

Reply:
`None` fails validation before reaching `_deserialize`.

    ValidationError: Field may not be null.